mempool/mempool.go

```diff
@@ -81,6 +81,10 @@ type Config struct {
 // Policy houses the policy (configuration parameters) which is used to
 // control the mempool.
 type Policy struct {
+	// TxVersion is the transaction version that the mempool should accept.
+	// All transactions above this version are rejected as non-standard.
+	TxVersion int32
```

Comment:
Great point, I've renamed the variable within the policy struct from `TxVersion` to `MaxTxVersion`.